File: struclin/queue/queue.cpp
#include <iostream> 
#include <queue> 
  
using namespace std; 
  
int main() 
{ 
    queue <int> myQueue;

    // push operation
    myQueue.push(1998);
    myQueue.push(2006);
    myQueue.push(2018);
    
    // pop operation
    myQueue.pop();

    // top operation
    cout << "\nfront of the queue is " << myQueue.front();

    // isEmpty operation
    cout << "\nqueue is empty ? " << myQueue.empty() << "\n";

    return 0; 
} 

File: struclin/queue/queue.py
from collections import deque 
  
myQueue = deque() 

# push operation
myQueue.append(1998) 
myQueue.append(2006) 
myQueue.append(2018) 

# top operation
print(myQueue[0])
  
# pop operation
print(myQueue.popleft())

# isEmpty operation
print(len(myQueue))